backend/FwLite/FwLiteWeb/install-launcher.sh

```
@@ -1,7 +1,10 @@
 #!/bin/bash
 
-# Get the full path to the current directory
-APPDIR="$(pwd)"
+# Get the full path to the script directory
+APPDIR="$(dirname "$(realpath "${BASH_SOURCE[0]}")")"
+
+# Make it the current directory so rest of script will work
+cd "$APPDIR"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)